receke/database/db.json

```diff
@@ -216,34 +188,43 @@
       "ingredients": [
         {
           "ingredient": "Puerro",
-          "quantity": "3-4 (grandes)"
+          "quantity": "3-4 (grandes)",
+          "image": "http://localhost:5173/src/assets/media/puerro.png"
```

Comment:
Images of the ingredients should not be here, they are in the ingredients array at the end of the document